packages/commons/src/auth/authenticationMiddleware.ts

```diff
@@ -20,80 +22,34 @@ export const authenticationMiddleware: (
 ) => ZodiosRouterContextRequestHandler<ExpressContext> =
   (config: JWTConfig) =>
   async (req, res, next): Promise<unknown> => {
-    const jwksClients = getJwksClients(config);
-
-    const addCtxAuthData = async (
-      authHeader: string,
-      logger: Logger
-    ): Promise<void> => {
-      const authorizationHeader = authHeader.split(" ");
-      if (
-        authorizationHeader.length !== 2 ||
-        authorizationHeader[0] !== "Bearer"
-      ) {
-        logger.warn(
-          `No authentication has been provided for this call ${req.method} ${req.url}`
-        );
-        throw missingBearer;
-      }
-
-      const jwtToken = authorizationHeader[1];
-      const valid = await verifyJwtToken(jwtToken, jwksClients, config, logger);
-      if (!valid) {
-        throw unauthorizedError("Invalid token");
-      }
-
-      const authData: AuthData = readAuthDataFromJwtToken(jwtToken, logger);
-      // eslint-disable-next-line functional/immutable-data
-      req.ctx.authData = authData;
-      next();
-    };
-
+    // We assume that:
+    // - contextMiddleware already set ctx.serviceName and ctx.correlationId
     const loggerInstance = logger({
       serviceName: req.ctx.serviceName,
       correlationId: req.ctx.correlationId,
     });
 
     try {
-      const headers = Headers.safeParse(req.headers);
```

Comment:
Removing parsing of the correlationId from auth middleware since it's unrelated - leaving only auth header parsing and token verification